myproject/.pre-commit-config.yaml

```diff
@@ -1,11 +1,11 @@
 repos:
 -   repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: main
+    rev: v4.5.0
     hooks:
     -   id: trailing-whitespace
     -   id: end-of-file-fixer
 -   repo: https://github.com/ecugol/pre-commit-hooks-django
-    rev: main
+    rev: v0.4.0
     hooks:
     -   id: check-unapplied-migrations
     -   id: check-absent-migrations
```

Comment:
Remove all the hooks. Include the following hooks:
1. black formatter: To format the code
2. isort: To sort all the import
3. Trailing whitespace: To remove unnecessary whitespaces that you already have